sdk/core/azure-core/azure/core/pipeline/__init__.py

```diff
@@ -95,7 +95,7 @@ def clear(self):  # pylint: disable=docstring-missing-return, docstring-missing-
         """
         raise TypeError("Context objects cannot be cleared.")
 
-    def update(self, *args, **kwargs):  # pylint: disable=docstring-missing-return, docstring-missing-rtype
+    def update(self, *args, **kwargs):  # pylint: disable=docstring-missing-return, docstring-missing-rtype, docstring-missing-param
```

Comment:
I have https://github.com/Azure/azure-sdk-for-python/pull/31018/files#diff-912d3049bc5b0f52aaf4e475a52c0576ee80a998d5a37658e2e6369435652cb5, which one should we use?

Reply:
Yours! Let me close this, sorry I didn't see that PR :)